mmv1/third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -393,6 +394,13 @@ func resourceComposerEnvironment() *schema.Resource {
 										ForceNew:    true,
 										Description: `The CIDR block from which IP range in tenant project will be reserved for Cloud SQL. Needs to be disjoint from web_server_ipv4_cidr_block.`,
 									},
+									"cloud_composer_network_ipv4_cidr_block": {
```

Comment:
The error you are running into locally is because `cloud_composer_network_ipv4_cidr_block` is not set in the api call. 

You will need to set cloud_composer_network_ipv4_cidr_block on request and set it on the config within the read. 

Check the `private_environment_config` expander and flattener